ingest/tutorial/example_statistics.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stellar/go/ingest"
+	backends "github.com/stellar/go/ingest/ledgerbackend"
+	"github.com/stellar/go/support/log"
+)
+
+var (
+	config = backends.CaptiveCoreConfig{
+		// Change these based on your env:
+		BinaryPath:        "/usr/local/bin/stellar-core",
+		ConfigAppendPath:  "stellar-core-stub.toml",
+		NetworkPassphrase: "Test SDF Network ; September 2015",
+		HistoryArchiveURLs: []string{
+			"https://history.stellar.org/prd/core-testnet/core_testnet_001",
+		},
+	}
+)
+
+func main() {
+	lg := log.New()
+	lg.SetLevel(logrus.InfoLevel)
+	lg.Entry.Logger.Out = ioutil.Discard
+	config.Log = lg
+
+	backend, err := backends.NewCaptive(config)
+	panicIf(err)
+	defer backend.Close()
+
+	// Prepare a range to be ingested:
+	var startingSeq uint32 = 2 // can't start with genesis ledger
+	var ledgersToRead uint32 = 10000
+
+	fmt.Printf("Preparing range (%d ledgers)... ", ledgersToRead)
+	ledgerRange := backends.BoundedRange(startingSeq, startingSeq+ledgersToRead)
+	err = backend.PrepareRange(ledgerRange)
+	panicIf(err)
+	fmt.Println("done.")
+
+	// These are the statistics that we're tracking.
+	var successfulTransactions, failedTransactions int
+	var operationsInSuccessful, operationsInFailed int
+
+	var delta uint32 = 0
+	for ; delta <= ledgersToRead; delta++ {
+		fmt.Printf("Processed %d/%d ledgers (%0.1f%%)... ",
+			delta, ledgersToRead, 100*float32(delta)/float32(ledgersToRead))
```

Comment:
I think it's overcomplicated for an example and distract reader from the essence.
```suggestion
		fmt.Printf("Processing %d... ", seq)
```